README.md

```diff
@@ -79,6 +79,9 @@ correctly.
 pebble -config ./test/config/pebble-config.json
 ```
 
+Note that the CA's root certificate is regenerated on every launch. It can be
+retrieved via `https://localhost:14000/root`.
```

Comment:
@felixfontein Could you expand on this section briefly to mention:
1. When you might need the root (such as for an end-to-end test of the generated certificates)
2. That adding the Pebble CA certificate to a trust store should only be done on **testing** systems.

For 2. you might find the "Root Certificate" text on this page helpful to crib from: https://letsencrypt.org/docs/staging-environment/

Reply:
It might also be worth emphasizing that the Pebble root CA != the Pebble HTTPS API certificate's root CA.

Reply:
I moved the text to a new section at the end of the document. I've also tried to adjust it as you requested.